Containers/nextcloud/Dockerfile

```
@@ -250,7 +250,8 @@ RUN set -ex; \
     chmod +x /cron.sh && \
     chmod +x /notify.sh && \
     chmod +x /notify-all.sh && \
-    chmod +x /activate-collabora.sh
+    chmod +x /activate-collabora.sh && \
+    chmod +x healthcheck.sh
```

why not also?
```suggestion
    chmod +x /healthcheck.sh
```
And does the file even get copied?

---

> why not also?

Typo

> And does the file even get copied?

Yes